vast.conf

```diff
@@ -306,19 +306,22 @@ logger {
 
   ; Configures the minimum severity of messages written to the log file.
   ; Possible values: quiet, error, warning, info, verbose, debug, trace.
-  ;file-verbosity = "debug"
+  ; File logging is only available for commands that start a node (e.g.,
+  ; vast start). The levels above 'verbose' are usually not available in
```

Comment:
Not sure, but intuitively I'd say `s/above/below/`?

Reply:
info -> low verbosity; debug -> high verbosity?

Reply:
Sure, but info -> higher importance, debug -> lower importance.

But I'll let you decide, this was only a side comment.